### ISSUE TITLE
Gladius USA Issues

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#127]
_From @kanelakis on April 18, 2017 13:47_

I also tested Gladius US version, setting the resolution in default, DIRECT3D REF (software), already at the launch comes the fateful message.
Add debug files.
[CxbxDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/929862/CxbxDebug.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/929863/KrnlDebug.txt)

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#377_

**Comment by kanelakis** [comment]
I will start doing the first tests and send videos and files.

**Comment by kanelakis** [comment]
I tested the game with these configurations, and with the version of the emulator Jun 14 2018.With the other settings gives me the corrupt or non-original .xbe file.
I attach images and the log file
[http://www.imagebam.com/image/803044896287604](url)
[http://www.imagebam.com/image/7a814f896287504](url)
[http://www.imagebam.com/image/355654896287384](url)
[http://www.imagebam.com/image/31d616896287314](url)

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2107557/KrnlDebug.txt)

**Comment by Voxel9** [comment]
Do not use D3D REF software for testing, you will just get a black screen with pretty much everything you test with it anyway.

If you use HAL (hardware) and keep LLE GPU off, there is the possibility of getting better results.

**Comment by kanelakis** [comment]
Ok, I wanted to do several tests to send more results, and to show the differences.

**Comment by kanelakis** [comment]
Test with version Jun 22 2018.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2129389/KrnlDebug.txt)

**Comment by Voxel9** [comment]
Does the game actually output any video or audio?

**Comment by kanelakis** [comment]
Test the game with version Jun 25 2018.
![2018-06-25 1](https://user-images.githubusercontent.com/26459477/41872689-f870be6c-78c2-11e8-9ede-ccdcb4d88da9.png)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2134545/KrnlDebug.txt)

**Comment by kanelakis** [comment]
Test the game with version Jul 1 2018,same as the previous post.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2153184/KrnlDebug.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2153186/KrnlDebug.txt)

**Comment by kanelakis** [comment]
Test the game with version Jul 13 2018.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2193240/KrnlDebug.txt)

**Comment by kanelakis** [comment]
Test the game with version Jul 19 2018.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2216549/KrnlDebug.txt)

**Comment by Voxel9** [comment]
Still black screen I’m assuming for both messages?

**Comment by kanelakis** [comment]
Yes, always black screen as the previous post, no news.If there will be changes with the next version place photos.

**Comment by kanelakis** [comment]
Test the game with version Jul 27 2018,I played a few tricks at the start of the emulator to see if anything changed, but nothing new.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2237530/KrnlDebug.txt)

**Comment by kanelakis** [comment]
Test the game with version Jul 28 2018,no news.
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2244665/KrnlDebug.txt)
I always use the basic settings, if there is another way let me know.

**Comment by Voxel9** [comment]
It's only been 3 days... no new commits have been added since. Why not wait to post until something new actually happens?

**Comment by kanelakis** [comment]
Ok,then I will post as soon as there will be changes.

**Comment by kanelakis** [comment]
[CxbxDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3861395/CxbxDebug.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/3861396/KrnlDebug.txt)
![2019-11-19 22](https://user-images.githubusercontent.com/26459477/69107235-fc13ea80-0a70-11ea-96ad-a8cdbc0cc9ab.png)
After a year I send a new update.

**Comment by kanelakis** [comment]
I updated my hardware and tried the December 2th version but nothing has changed, it always remains dark.
Intel core i9 9900k 3.60
Ram: 32 Gb
Videocard: Gigabyte GTX 1060 6gb.

**Comment by kanelakis** [comment]
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4324311/Xbe.txt)
I test the game with version Mar 9 2020

**Comment by kanelakis** [comment]
![2020-06-06 (2)](https://user-images.githubusercontent.com/26459477/83930126-5d45a600-a796-11ea-9773-e397e037f1be.png)
[CxbxDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4738989/CxbxDebug.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4738990/KrnlDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4738991/Xbe.txt)
